pop/POPAnimationTracer.mm

```diff
@@ -32,18 +32,19 @@ @implementation POPAnimationTracer
     : self->_animationState->lastTime;
 
   POPAnimationEvent *event;
+  __strong POPAnimation* animation = self->_animation;
```

Comment:
I really want to be explicit here as a reminder to not use weak ivar multiple times.